src/test/scala/com/amazon/deequ/analyzers/jdbc/JdbcFixtureTests.scala

```diff
@@ -0,0 +1,21 @@
+package com.amazon.deequ.analyzers.jdbc
+
+import org.scalatest.{Matchers, WordSpec}
+
+class JdbcFixtureTests
```

Comment:
Does not test fixtures, but a helper method for tests. Test code should not be tested

Reply:
Deleted